hoodie-client/src/test/java/HoodieClientExample.java

```diff
@@ -110,5 +112,15 @@ public void run() throws Exception {
     records.addAll(dataGen.generateUpdates(newCommitTime, 100));
     writeRecords = jsc.<HoodieRecord>parallelize(records, 1);
     client.upsert(writeRecords, newCommitTime);
+
+    /**
+     * Schedule a compaction and also perform compaction on a MOR dataset
+     */
+    if (tableType == HoodieTableType.MERGE_ON_READ.name()) {
```

Comment:
Sorry to PR snipe, but I'm just curious - how does the `==` work here if it's just an instance check on `String` (and not the `enum`?

Reply:
+1

Reply:
Fair point, it should work on the enum, changed it.